### PR TITLE
[WIP] Builder archetype (work)

### PR DESCRIPTION
**Pull request by chaseadamsio**
The initial update for using `builder` instead of `bolt` as the executable. #47 

@ryan-roemer

### CODE REVIEW

File: packages/electrode-bolt/package.json [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
   "scripts": {
-    "dev": "bolt server-dev & bolt server-test",
-    "hot": "bolt server-hot & bolt server-test",
+    "dev": "builder run server-dev & builder run server-test",
+    "hot": "builder run server-hot & builder run server-test",
     "lint-react-demo": "eslint --ext .js,.jsx -c ./node_modules/electrode-bolt/config/eslint/.eslintrc-react-demo demo/*.jsx --color",
     "lint-react-src": "eslint --ext .js,.jsx -c ./node_modules/electrode-bolt/config/eslint/.eslintrc-react src --color",
     "lint-react-test": "eslint --ext .js,.jsx -c ./node_modules/electrode-bolt/config/eslint/.eslintrc-react-test test/client --color",
-    "lint": "bolt lint-react-demo && bolt lint-react-src && bolt lint-react-test",
+    "lint": "builder run lint-react-demo && builder run lint-react-src && builder run lint-react-test",
```
Comment (ryan-roemer) [inline packages/electrode-bolt/package.json]: OPTIONAL: Could do `builder concurrent lint-react-demo lint-react-src lint-react-test` for speed.

File: packages/electrode-bolt/package.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
 {
   "name": "electrode-bolt",
   "version": "1.0.9",
-  "description": "Opinionated meta config runner for react components",
+  "description": "Builder Archetype - Electrode Component",
   "scripts": {
-    "dev": "bolt server-dev & bolt server-test",
-    "hot": "bolt server-hot & bolt server-test",
+    "dev": "builder run server-dev & builder run server-test",
```
Comment (ryan-roemer) [inline packages/electrode-bolt/package.json]: GLOBAL: Should refactor all background / concurrent tasks to, e.g.:

```
builder concurrent server-dev server-test
```

Comment (chaseadamsio) [inline packages/electrode-bolt/package.json]: great call out. Thanks for catching it!

File: packages/electrode-bolt/config/babel/.babelrc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (ryan-roemer) [inline packages/electrode-bolt/config/babel/.babelrc]: NOTE: babelrc _needs_ to be in root project and cannot be "pointed to" from archetype. @exogen has a better handle on what's up with this problem.

**Comment by ryan-roemer** [comment]
@chaseadamsio -- Few comments, rest LGTM. Haven't tested anything out though...

File: packages/electrode-bolt/config/babel/.babelrc (hunk 3 above)
Comment (exogen) [inline packages/electrode-bolt/config/babel/.babelrc]: Yup. We sorta have an issue open to track this, but it needs better documentation.

Quick explanation, some tools that include Babel support (`isparta-loader` is one example, but I think there are more) don't have an option to send it a config – they just check for `.babelrc` in a typical dir structure.

Comment (chaseadamsio) [inline packages/electrode-bolt/config/babel/.babelrc]: I added it there, but it should really be in `/templates`

**Comment by chaseadamsio** [comment]
Considering going ahead and bumping babel to @6.x.x since this is going to be a major breaking release. Thoughts? cc @baer @ryan-roemer

**Comment by ryan-roemer** [comment]
@boygirl @exogen @kenwheeler @alexlande -- Are we ready for babel6 yet in our victory repos?

@chaseadamsio -- I think best bet is "just focus on the scaffolding" to isolate the feature. Then _separately_ we tackle upgrades.

**Comment by chaseadamsio** [comment]
Fair point. No need for unneeded extra complexity!

**Comment by chaseadamsio** [comment]
I'm going to go through this commit one more time in the morning and double check, but I tested this in a project and everything seemed to work as it should.

**Pull request closed without merging**